applications/spacewalk-bridge.md

```diff
@@ -0,0 +1,231 @@
+
+- **Deposit**: bridge any Stellar asset to Substrate-based chains. The tokens of the source asset are transferred to some dedicated **bridge account** in Stellar where they are locked. The bridge then mints wrapped tokens in the target chain and transfers them to the recipient account.
+- **Withdrawal**: bridge wrapped tokens in Substrate-based chains to Stellar. First, the user instructs the bridge to burn wrapped tokens in the Substrate-based chain. Afterwards, the bridge unlocks tokens held in the bridge account in Stellar and transfers them to some target account.
+
+Stellar is not smart contract capable – therefore we follow the recommendation laid out in the [Polkadot documentation](https://wiki.polkadot.network/docs/learn-bridges#via-higher-order-protocols) and base our bridge design on XCLAIM. In XCLAIM, the bridge account is called a **vault**. XCLAIM is based on the following four core features:
+
+- Implement a chain relay for Stellar in the bridge pallet
+- Employ collateralization in order to ensure that the vault exhibits good behavior
+- Ensure that the economic value of the collateral exceeds the value of the vault
+- Enable a decentralized network of vaults
+
+Our bridge differs from XCLAIM in some details as follows:
+
+- Implementing a full chain relay for Stellar is out of scope of this web3 grant proposal as this requires an update to the Stellar consensus protocol. Instead the Spacewalk protocol assumes that a light Stellar node is co-located for every node of the Substrate-based chain and uses the Stellar nodes as an oracle.
+- We will employ a single vault instead of a network of vaults because according to Section V.E. of [Xclaim: Trustless, interoperable, cryptocurrency-backed assets](https://eprint.iacr.org/2018/643.pdf), the latter version leads to non-fungible wrapped tokens which is not a desirable condition.
+
+**Architecture**
+
+![Stellar Bridge Web3 Grant(3)](https://user-images.githubusercontent.com/52105313/146759286-01810328-a383-4306-9ad2-61515913b7fb.png)
+
+The architecture of the bridge consists of the following components:
+
+- **Bridge accounts**: this is a set of escrow accounts used to lock assets. They are completely and only controlled by the bridge nodes. The Spacewalk bridge potentially requires multiple bridge accounts instead of a single bridge accounts because in Stellar a single account is limited to hold up to 1000 distinct assets. Every bridge account is associated with a maximum of 1000 distinct Stellar assets and will be used whenever one of its associated assets is used in a deposit or withdrawal operation. Stellar users initiate a deposit by sending tokens to the appropriate bridge account, which they request from the bridge pallet prior to the deposit. Likewise bridge nodes will instruct the bridge accounts to unlock and send tokens back the users during a withdrawal.
```

Comment:
are these escrow accounts / bridge nodes operated by distinct actors? if so, how does this interplay with having "a single vault"?